.github/renovate.json5

```diff
@@ -68,7 +67,7 @@
     "main",
     "v1.16",
     "v1.15",
-    "v1.14",
+    "v1.14"
```

Comment:
nit: json5 allows the existing syntax with a comma at the end of an entry in a list, which is nice as it means we don't have to modify unrelated lines when making changes like this.

Reply:
All similar changes should not be necessary.